mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -242,7 +243,9 @@ def action_manual_check_mailgun(self):
                 raise ValidationError(_("Event information not longer stored"))
             for item in content["items"]:
                 if not self.env['mail.tracking.event'].search(
-                        [('mailgun_id', '=', item["id"])]):
+                        [('mailgun_id', '=', item["id"])]) and (
+                            item.get("recipient", "") ==
+                            email_split(tracking.recipient)[0]):
```

Comment:
:thinking: I'd really appreciate if you can rewrite this `if` in a more readable fashion, with a little comment that explains what is it doing... I've been staring them for awhile and I still don't understand them :nerd_face:

Reply:
you still approved, does this mean you don't care so much or should I wait with merging?

Reply:
I feel it was more a matter of taste, but I didn't want to block it for such.

Thanks